.gitlab/e2e/e2e.yml

```diff
@@ -139,7 +139,7 @@ new-e2e-agent-subcommands:
       - EXTRA_PARAMS: --run "Test(Linux|Windows)StatusSuite"
       - EXTRA_PARAMS: --run "Test(Linux|Windows)HealthSuite"
       - EXTRA_PARAMS: --run "Test(Linux|Windows)ConfigSuite"
-      - EXTRA_PARAMS: --run "Test(Linux|Windows|LinuxAzure)HostnameSuite"
+      - EXTRA_PARAMS: --run "Test(Linux|Windows|LinuxAzure|LinuxGCP)HostnameSuite"
```

Comment:
What about making the regex take all Linux tests with `Linux.*`? 
`--run "Test(Linux.*|Windows)HostnameSuite"`

Reply:
I like the fact that the suites are explicitly listed, you can't accidentally run something, while if you start introducing `.*` you could run more than intended...